packages/core/src/tegel-light/components/tl-datetime/tl-datetime.scss

```diff
@@ -0,0 +1,282 @@
+@import './tl-datetime-vars';
+@import '../../../mixins/box-sizing';
+@import '../../../../../../typography/utilities/typography-utility';
+
+.tl-datetime {
+  display: block;
+  min-width: 208px;
+  background: unset;
+
+  &--nomin {
+    min-width: auto;
+  }
+
+  /* Native picker visuals */
+  input[type='datetime-local'],
+  input[type='date'],
+  input[type='month'],
+  input[type='week'],
+  input[type='time'] {
+    &::-webkit-inner-spin-button,
+    &::-webkit-calendar-picker-indicator {
+      opacity: 0;
+    }
+  }
+}
+
+/* Firefox-specific icon hiding for date/datetime */
+@supports (-moz-appearance: none) {
+  .tl-datetime__icon--datetime {
+    .tl-datetime.type-datetime-local &,
+    .tl-datetime.type-date & {
+      display: none;
+    }
+  }
+}
+
+.tl-datetime__label {
+  @include detail-05;
+
+  display: block;
+  margin-bottom: var(--tds-spacing-element-8);
+  color: var(--datetime-label);
+
+  .tl-datetime--disabled & {
+    color: var(--datetime-label-disabled);
+    cursor: not-allowed;
+  }
+}
+
+.tl-datetime__container {
+  border-radius: var(--datetime-border-radius);
+  position: relative;
+  height: 56px;
+  box-sizing: border-box;
+  background-color: var(--datetime-background);
+  transition: border-bottom-color 200ms ease;
+  border-left: 1px solid var(--datetime-border-color-left);
+  border-right: 1px solid var(--datetime-border-color-right);
+  border-bottom: 1px solid var(--datetime-border-color-bottom);
+  border-top: 1px solid var(--datetime-border-color-top);
+
+  &:hover {
+    border-left: 1px solid var(--datetime-border-color-left-hover);
+    border-right: 1px solid var(--datetime-border-color-right-hover);
+    border-bottom: 1px solid var(--datetime-border-color-bottom-hover);
+    border-top: 1px solid var(--datetime-border-color-top-hover);
+  }
+
+  .tl-datetime--md & {
+    height: 48px;
+  }
+
+  .tl-datetime--sm & {
+    height: 40px;
+  }
+
+  .tl-datetime--focus & {
+    border-left: 1px solid var(--datetime-border-color-left-focus);
+    border-right: 1px solid var(--datetime-border-color-right-focus);
+    border-bottom: 1px solid var(--datetime-border-color-bottom-focus);
+    border-top: 1px solid var(--datetime-border-color-top-focus);
+    outline: 2px solid var(--datetime-outline-color-focus);
+    outline-offset: -1px;
+  }
+
+  .tl-datetime--disabled & {
+    cursor: not-allowed;
+    border-color: transparent;
+  }
+
+  .tl-datetime--state-success & {
```

Comment:
Do we need to specify that its a `state` here? instead just do `--success`? Sanity check, what do you think @NordbergAnna ?